specification/draft/apps.mdx

```diff
@@ -101,37 +190,6 @@ The resource content is returned via `resources/read`:
 - Content MUST be provided via either `text` (string) or `blob` (base64-encoded)
 - Content MUST be valid HTML5 document
 
-#### Metadata Fields:
```

Comment:
Same question as in https://github.com/modelcontextprotocol/ext-apps/pull/3 - perhaps it's easier to have the fields descriptions grouped - for easy skimming?
I agree that in any case we need to maintain consistency across the document